### ISSUE TITLE
"function () { [bytecode] }" is not a valid color or brush

### ISSUE THREAD

**Issue by enchorb** [marklawlor/nativewind#675]
Getting this on any version above `4.0.13`

**app.tsx**

```
vars({
'--color-primary-50': '240, 253, 250'
'--color-primary-100': '204, 251, 241'
...
})
```

**tailwind.config.js**

```
/** @type {import('tailwindcss').Config} */
const { createGlobPatternsForDependencies } = require('@nx/react/tailwind');
const { join } = require('path');
const defaultTheme = require('tailwindcss/defaultTheme');

module.exports = {
  darkMode: 'class',
  presets: [require('nativewind/preset')],
  content: [
    join(__dirname, '{src,app,pages,components}/**/!(*.stories|*.spec).{js,jsx,ts,tsx,html}'),
    ...createGlobPatternsForDependencies(__dirname)
  ],
  theme: {
    extend: {
      colors: {
        primary: {
          50: 'rgb(var(--color-primary-50) / <alpha-value>)',
          100: 'rgb(var(--color-primary-100) / <alpha-value>)',
          200: 'rgb(var(--color-primary-200) / <alpha-value>)',
          300: 'rgb(var(--color-primary-300) / <alpha-value>)',
          400: 'rgb(var(--color-primary-400) / <alpha-value>)',
          500: 'rgb(var(--color-primary-500) / <alpha-value>)',
          600: 'rgb(var(--color-primary-600) / <alpha-value>)',
          700: 'rgb(var(--color-primary-700) / <alpha-value>)',
          800: 'rgb(var(--color-primary-800) / <alpha-value>)',
          900: 'rgb(var(--color-primary-900) / <alpha-value>)'
        }
      }
    }
  },
  plugins: [require('@tailwindcss/typography'), require('tailwindcss-animate')]
};
```

**Comment by enchorb** [comment]
@marklawlor any idea on the change that chased this?

**Comment by fb55** [comment]
We are getting this error when using `nativeStyleToProp` for a prop that uses a color variable.

**Comment by marklawlor** [comment]
This is fixed in `4.0.30`
